### DISCUSSION TITLE
Export as Word file without pictures

### DISCUSSION THREAD

**Discussion by condorheroblog** [hunghg255/reactjs-tiptap-editor#35] [General]
When rich text editor export to Word uses `prosemirror-docx` repository, which is a server-side open source repository and cannot be run in the browser.

Here is the reason: `prosemirror-docx` uses the `sizeOf` API of `buffer-image-size` to measure the width and height of the image when exporting images. `buffer-image-size` directly uses the `Buffer` object, which belongs to `Node.js` and does not exist in the browser.

Look this picture, it's good.

<img width="691" alt="image" src="https://github.com/user-attachments/assets/39dd2ce1-ea27-4636-bcea-c30d8b38846a">

How to fix this bug? Use the browser API to implement it. I recommended image-dimensions.

https://www.npmjs.com/package/image-dimensions

What's your opinion?👋

**Comment by hunghg255** [comment]
@condorheroblog oh look so great!

**Comment by hunghg255** [comment]
@condorheroblog can you create PR?

**Reply by condorheroblog** [reply]
OK, I have tried out it😂.

https://github.com/curvenote/prosemirror-docx/pull/35

**Reply by hunghg255** [reply]
> OK, I have tried out it😂.
> 
> [curvenote/prosemirror-docx#35](https://github.com/curvenote/prosemirror-docx/pull/35)

great!
